recipes-kernel/linux/linux-fslc-imx/mx9-nxp-bsp/defconfig

```diff
@@ -0,0 +1,1071 @@
+CONFIG_SYSVIPC=y
```

Comment:
What is the purpose of the default `defconfig` for all `mx9`? At the least the commit should explain why we add it, but perhaps even better would be to not add it until it is actually used?

Reply:
I'd let the `mx93` one, only.

Reply:
I will fix the commit log, but they are the same defconfig, but one for `linux-imx`and one for `linux-fslc-imx`. It happens that today they are the same.
Is there a better way to have one file only for both linux?

Reply:
Ah, I did not notice the two files are for different linux. For `linux-imx`, I see that this is the same `defconfig` as for all i.MX 8. Perhaps we can link `linux-imx/imx9-nxp-bsp/defconfig` to `linux-imx/imx8-nxp-bsp/defconfig`? Or better, can we replace both with `linux-imx/aarch64/defconfig`?

Reply:
Using `aarch64` seems like a good option.

Reply:
then using `aarch64` the `imx-nxp-bsp` is avoided which causes an error for all the machines, so I suggest we use `imx8-nxp-bsp` and `imx9-nxp-bsp`

Reply:
Can we change `imx-nxp-bsp` to `arm`?